tests/grid/grid_generator_subdivided_hyper_cube_01.cc

```diff
@@ -0,0 +1,56 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 - 2018 by the deal.II authors
```

Comment:
You could also fix the year here.

Reply:
I applied all your comments. Might as well do things correctly the first time :)! Please tell me if you feel anything else should be changed.